modules/administration-guide/partials/proc_collecting-dev-workspace-engine-metrics-with-prometheus.adoc

```diff
@@ -0,0 +1,62 @@
+[id="proc_collecting-dev-workspace-engine-metrics-with-prometheus_{context}"]
+= Collecting {devworkspace} engine metrics with Prometheus
+
+[role="_abstract"]
+This section describes how to use the Prometheus to collect, store, and query metrics about the {devworkspace} engine.
+
+.Prerequisites
+
+* The `devworkspace-controller-metrics` service is exposing metrics on port `8443`.
+
+* The `devworkspace-webhookserver` service is exposing metrics on port `9443`.
```

Comment:
this is going to be always the case right, or is it possible to opt out from metrics exposure

Reply:
I noticed that the [`devworkspace-controller-metrics`](https://github.com/devfile/devworkspace-operator/blob/main/deploy/deployment/kubernetes/objects/devworkspace-controller-metrics.Service.yaml) service is missing in some cases (ie. installation on minikube with chectl). Related issue: https://github.com/eclipse/che/issues/20762.